libs/react-components/src/community/components/layout/sidenav/sidenav.module.scss

```diff
@@ -35,18 +35,89 @@
 .sidenav__list {
   margin: 0;
   list-style: none;
+
+  .sidenav__list {
+    margin: 0;
+    background: var(--color-primary-highlight);
+  }
 }
 
 .sidenav__item {
   display: flex;
   flex-direction: row;
-  border-top: 1px solid var(--color-primary-active);
 
-  &:last-of-type {
-    border-bottom: 1px solid var(--color-primary-active);
+  .sidenav--has-dividers & {
+    border-top: 1px solid var(--color-primary-active);
+
+    &:last-of-type {
+      border-bottom: 1px solid var(--color-primary-active);
+    }
+  }
+
+  & [data-name='collapse'] {
+    width: 100%;
+  }
+
+  & [data-name='collapse-content'] {
+    margin: 0;
+  }
+
+  & [data-name='collapse-trigger'] {
+    &[aria-expanded='true'],
+    &:hover,
+    &.sidenav__item--current {
+      background-color: var(--color-primary-active);
+    }
+
+    [class^='sidenav__link'] {
+      padding-right: 0;
+    }
+
+    [data-name='icon'] {
+      color: var(--color-text-inverted);
+    }
+  }
+
+  & div[class^='collapse__content'] {
+    margin: 0;
+  }
+
+  & span[class^='row'] {
+    margin-right: 0;
+    margin-left: 0;
+
+    & span[class^='col']:first-child {
+      padding-right: 0;
+      padding-left: 0;
+    }
+
+    & span[class^='col']:last-child {
+      padding-left: 0;
+    }
   }
 }
 
+.sidenav__item .sidenav__item {
+  [data-name='collapse-trigger'] {
```

Comment:
Focus-visible styles for the collapse trigger are not the greatest 
![image](https://github.com/user-attachments/assets/563bbd63-cba9-40b8-affd-4acf6da04adb)